diff -ruN A/lib/hcc-config.cmake.in B/lib/hcc-config.cmake.in
--- A/lib/hcc-config.cmake.in	2019-10-09 15:06:50.204447194 -0400
+++ B/lib/hcc-config.cmake.in	2019-10-09 15:09:08.898036445 -0400
@@ -6,21 +6,24 @@
 
 find_path(HSA_HEADER hsa/hsa.h
   PATHS
-    /opt/rocm/include
+    @rocr@/include
 )
 
 find_library(HSA_LIBRARY hsa-runtime64
   PATHS
-    /opt/rocm/lib
+    @rocr@/lib
 )
 
-add_library(hsa-runtime64 SHARED IMPORTED)
+if (NOT HSA_FOUND)
+  add_library(hsa-runtime64 SHARED IMPORTED)
 
-set_target_properties(hsa-runtime64 PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${HSA_HEADER}"
-  IMPORTED_LOCATION "${HSA_LIBRARY}"
-  INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
-)
+  set_target_properties(hsa-runtime64 PROPERTIES
+    INTERFACE_INCLUDE_DIRECTORIES "${HSA_HEADER}"
+    IMPORTED_LOCATION "${HSA_LIBRARY}"
+    INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
+  )
+  set(HSA_FOUND YES)
+endif()  
 
 include( "${CMAKE_CURRENT_LIST_DIR}/ImportedTargets.cmake" )
 include( "${CMAKE_CURRENT_LIST_DIR}/hcc-targets.cmake" )
