diff --exclude '*~' -ruN A/lib/mcwamp.cpp B/lib/mcwamp.cpp
--- A/lib/mcwamp.cpp	2019-11-13 15:44:18.683425447 -0500
+++ B/lib/mcwamp.cpp	2019-11-13 15:46:12.588421527 -0500
@@ -157,7 +157,7 @@
  */
 class HSAPlatformDetect : public PlatformDetect {
 public:
-  HSAPlatformDetect() : PlatformDetect("HSA", get_library_path() + LIB_NAME_WITH_VERSION("libmcwamp_hsa.so"), nullptr) {}
+  HSAPlatformDetect() : PlatformDetect("HSA", "@out@/lib/libmcwamp_hsa.so", nullptr) {}
 };
 
 
@@ -171,7 +171,7 @@
   // load HSA C++AMP runtime
   if (mcwamp_verbose)
     std::cout << "Use HSA runtime" << std::endl;
-  std::string lib = get_library_path() + LIB_NAME_WITH_VERSION("libmcwamp_hsa.so");
+  std::string lib = "@out@/lib/libmcwamp_hsa.so";
   runtimeImpl = new RuntimeImpl(lib.c_str());
   if (!runtimeImpl->m_RuntimeHandle) {
     std::cerr << "Can't load HSA runtime!" << std::endl;
@@ -188,7 +188,7 @@
   // load CPU runtime
   if (mcwamp_verbose)
     std::cout << "Use CPU runtime" << std::endl;
-  std::string lib = get_library_path() + LIB_NAME_WITH_VERSION("libmcwamp_cpu.so"); 
+  std::string lib = "@out@/lib/libmcwamp_cpu.so"; 
   runtimeImpl = new RuntimeImpl(lib.c_str());
   if (!runtimeImpl->m_RuntimeHandle) {
     std::cerr << "Can't load CPU runtime!" << std::endl;
