diff -ruN --exclude '*~' A/CMakeLists.txt B/CMakeLists.txt
--- A/CMakeLists.txt	2020-06-10 16:24:12.447045228 -0400
+++ B/CMakeLists.txt	2020-06-10 16:25:41.406397581 -0400
@@ -12,27 +12,6 @@
 
 rocm_setup_version(VERSION 0.5.0)
 
-if(CMAKE_CXX_COMPILER MATCHES ".*hcc")
-    set(HCC_EXE ${CMAKE_CXX_COMPILER})
-else()
-    find_program(HCC_EXE NAMES hcc clang PATHS /opt/rocm/hcc/bin)
-endif()
-
-get_filename_component(HCC_BIN ${HCC_EXE} DIRECTORY)
-if(NOT HCC_BIN)
-    message("HCC_EXE: ${HCC_EXE}")
-    message("HCC_BIN: ${HCC_BIN}")
-    message(FATAL_ERROR "Can't find hcc")
-endif()
-
-set(CLANG_BIN "${HCC_BIN}" CACHE STRING "")
-set(BITCODE_DIR "${HCC_BIN}/../lib" CACHE STRING "")
-
-message("")
-message("--------CLANG_BIN: ${CLANG_BIN}")
-message("------BITCODE_DIR: ${BITCODE_DIR}")
-message("")
-
 set(TRIPLE amdgcn-amd-amdhsa)
 message(STATUS "Triple: ${TRIPLE}")
 
