diff --exclude '*~' -ruN A/caffe2/CMakeLists.txt B/caffe2/CMakeLists.txt
--- A/caffe2/CMakeLists.txt     2019-11-26 20:36:56.803756325 -0500
+++ B/caffe2/CMakeLists.txt     2019-11-26 20:40:57.716081278 -0500
@@ -878,7 +878,7 @@
 # NB: This must be target_compile_definitions, not target_compile_options,
 # as the latter is not respected by nvcc
 target_compile_definitions(torch PRIVATE "-DCAFFE2_BUILD_MAIN_LIB")
-
+target_compile_definitions(torch PUBLIC HIP_VERSION=2)

 # ATen parallelism settings
 #  OMP - OpenMP for intra-op, native thread pool for inter-op parallelism
