diff --exclude '*~' -ruN a/third_party/gpus/rocm_configure.bzl b/third_party/gpus/rocm_configure.bzl      
--- a/third_party/gpus/rocm_configure.bzl
+++ b/third_party/gpus/rocm_configure.bzl
@@ -186,58 +186,60 @@
       host compiler include directories, which can be added to the CROSSTOOL
       file.
     """
-    inc_dirs = []
+    #inc_dirs = []
 
     # general ROCm include path
-    inc_dirs.append(rocm_config.rocm_toolkit_path + "/include")
+    #inc_dirs.append(rocm_config.rocm_toolkit_path + "/include")
 
     # Add HSA headers
-    inc_dirs.append("/opt/rocm/hsa/include")
+    #inc_dirs.append("/opt/rocm/hsa/include")
+    #inc_dirs.append("/opt/rocm/hsa/include/hsa")
 
     # Add HIP headers
-    inc_dirs.append("/opt/rocm/include/hip")
-    inc_dirs.append("/opt/rocm/include/hip/hcc_detail")
-    inc_dirs.append("/opt/rocm/hip/include")
+    #inc_dirs.append("/opt/rocm/include/hip")
+    #inc_dirs.append("/opt/rocm/include/hip/hcc_detail")
+    #inc_dirs.append("/opt/rocm/hip/include")
 
     # Add HIP-Clang headers
-    inc_dirs.append("/opt/rocm/llvm/lib/clang/8.0/include")
-    inc_dirs.append("/opt/rocm/llvm/lib/clang/9.0.0/include")
-    inc_dirs.append("/opt/rocm/llvm/lib/clang/10.0.0/include")
+    #inc_dirs.append("/opt/rocm/llvm/lib/clang/8.0/include")
+    #inc_dirs.append("/opt/rocm/llvm/lib/clang/9.0.0/include")
+    #inc_dirs.append("/opt/rocm/llvm/lib/clang/10.0.0/include")
 
     # Add rocrand and hiprand headers
-    inc_dirs.append("/opt/rocm/rocrand/include")
-    inc_dirs.append("/opt/rocm/hiprand/include")
+    #inc_dirs.append("/opt/rocm/rocrand/include")
+    #inc_dirs.append("/opt/rocm/hiprand/include")
 
     # Add rocfft headers
-    inc_dirs.append("/opt/rocm/rocfft/include")
+    #inc_dirs.append("/opt/rocm/rocfft/include")
 
     # Add rocBLAS headers
-    inc_dirs.append("/opt/rocm/rocblas/include")
+    #inc_dirs.append("/opt/rocm/rocblas/include")
 
     # Add MIOpen headers
-    inc_dirs.append("/opt/rocm/miopen/include")
+    #inc_dirs.append("/opt/rocm/miopen/include")
 
     # Add RCCL headers
-    inc_dirs.append("/opt/rocm/rccl/include")
+    #inc_dirs.append("/opt/rocm/rccl/include")
 
     # Add hcc headers
-    inc_dirs.append("/opt/rocm/hcc/include")
-    inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/7.0.0/include/")
-    inc_dirs.append("/opt/rocm/hcc/lib/clang/7.0.0/include")
+    #inc_dirs.append("/opt/rocm/hcc/include")
+    #inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/7.0.0/include/")
+    #inc_dirs.append("/opt/rocm/hcc/lib/clang/7.0.0/include")
 
     # Newer hcc builds use/are based off of clang 8.0.0.
-    inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/8.0.0/include/")
-    inc_dirs.append("/opt/rocm/hcc/lib/clang/8.0.0/include")
+    #inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/8.0.0/include/")
+    #inc_dirs.append("/opt/rocm/hcc/lib/clang/8.0.0/include")
 
     # Support hcc based off clang 9.0.0, included in ROCm2.2
-    inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/9.0.0/include/")
-    inc_dirs.append("/opt/rocm/hcc/lib/clang/9.0.0/include")
+    #inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/9.0.0/include/")
+    #inc_dirs.append("/opt/rocm/hcc/lib/clang/9.0.0/include")
 
     # Support hcc based off clang 10.0.0, included in ROCm2.8
-    inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/10.0.0/include/")
-    inc_dirs.append("/opt/rocm/hcc/lib/clang/10.0.0/include")
+    #inc_dirs.append("/opt/rocm/hcc/compiler/lib/clang/10.0.0/include/")
+    #inc_dirs.append("/opt/rocm/hcc/lib/clang/10.0.0/include")
 
-    return inc_dirs
+    #return [str(repository_ctx.path(spath).realpath) for spath in inc_dirs]
+    return nixos sed target
 
 def _enable_rocm(repository_ctx):
     if "TF_NEED_ROCM" in repository_ctx.os.environ:
